docs/user/guides/setup/git-repo-manual.rst

```diff
@@ -125,6 +131,19 @@ As an example, the URL pattern looks like this: ``https://readthedocs.org/api/v2
 Use this URL when setting up a new integration with your provider ^^ these steps vary depending on the provider.
 
 
+.. warning::
+
+   :doc:`/guides/pull-requests`
+      Git repositories that are imported manually **do not** have the required setup to send back a **commit status**.
+      If you need this integration,
+      you have to :doc:`configure the repository automatically </guides/setup/git-repo-automatic>`.
```

Comment:
Why are we linking to the PR guide in a warning? It seems confusing, we should just explain that PR builds don't work?

Reply:
Agreed!